### PR TITLE
[BACKLOG-34593] New MapR6.1 shim for pentaho 9.2

### PR DESCRIPTION
**Pull request by kodehash**

### CODE REVIEW

File: shims/mapr61/pmr/pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <version>9.2.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>pentaho-hadoop-shims-mapr61-pmr-libs</artifactId>
+
+    <version>9.2.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <properties>
+        <org.apache.hbase.version>2.1.0-cdh6.1.0</org.apache.hbase.version>
```
Comment (lucboudreau) [inline shims/mapr61/pmr/pom.xml:17]: This doesn't look right.

**Review by lucboudreau — CHANGES_REQUESTED** [review]
Only found one issue. Otherwise looks good.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 12m 4s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
shims,shims/mapr61,shims/mapr61/driver,shims/mapr61/kar,shims/mapr61/pmr
```
---
:ok_hand: All tests passed!
 
Tests run: **30**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1180/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 10m 42s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
shims,shims/mapr61,shims/mapr61/driver,shims/mapr61/kar,shims/mapr61/pmr
```
---
:ok_hand: All tests passed!
 
Tests run: **30**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1180/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by lucboudreau — APPROVED** [review]

**Pull request merged**